tutorials/howtos/notFoundPage.md

```diff
@@ -6,7 +6,7 @@ layout: tutorial
 To get 404 not found pages to display (instead of an
 error saying you don't have one), just create:
 
-```
+```bash
```

Comment:
This also isn't bash, but just a file path. If it makes it look nicer that's fine though.